include/nbl/asset/IMeshPacker.h

```diff
@@ -305,29 +258,119 @@ class IMeshPacker : public MeshPackerBase
         core::vector<Triangle> triangles;
     };
 
-    virtual core::vector<TriangleBatch> constructTriangleBatches(MeshBufferType& meshBuffer) = 0;
+    core::vector<TriangleBatch> constructTriangleBatches(MeshBufferType& meshBuffer)
+    {
+        const size_t idxCnt = meshBuffer.getIndexCount();
+        const uint32_t triCnt = idxCnt / 3;
+        _NBL_DEBUG_BREAK_IF(idxCnt % 3 != 0);
```

Comment:
extend this function so that it can convert other topologies of triangles (c.f. `homogenizePrimitiveTypeAndIndices`)

Reply:
TODO